docs/deploy/create_server.rst

```diff
@@ -202,6 +202,115 @@ Create the server via the :ref:`host<hosting>`'s interface.
             -  `Windows Server 2019 <https://docs.hetzner.com/robot/dedicated-server/windows-server/windows-server-2019/>`__
             -  `Installing Windows without KVM <https://community.hetzner.com/tutorials/install-windows>`__
 
+   .. tab-item:: Azure
+      :sync: azure
+
+      #. `Log into Azure <https://portal.azure.com>`__
+      #. Click the *Virtual machines* icon
+      #. Click the *Create* menu
+      #. Click the *Azure virtual machine* menu item
+
+         #. Set *Subscription* to "Microsoft Azure Sponsorship (4e98b5b1-1619-44be-a38e-90cdb8e4bc95)"
+         #. Set *Resource group* to the appropriate group (e.g. ``kingfisher``), creating a new resource group, if needed
+         #. Set *Virtual machine name* to the server's FQDN (e.g. ``ocp42.open-contracting.org``)
+         #. Set *Region* to "(Europe) UK South"
+         #. Set *Image* to the latest Ubuntu LTS version
+         #. Set *Size* to the appropriate size (e.g. ``B2s``)
+         #. Set *Authentication type* to "Password"
+         #. Set *Username* to "ocpadmin"
+         #. Set *Password* to a `strong password <https://www.lastpass.com/features/password-generator>`__
+
+      #. Click the *Next : Disks >* button
+
+         #. Set *OS disk type* to *Standard SSD*
+         #. Add additional disks, if appropriate:
+
+            #. Click the *Create and attach a new disk* link
+            #. Click the *Change size* link
+            #. Select "Standard SSD" from the *Storage type* dropdown
+            #. Click the desired size
+            #. Click the *OK* button
+
+      #. Click the *Next : Networking >* button
+
+         #. Set *Virtual network* to an appropriate name with a ``-vnet`` suffix (e.g. ``kingfisher-vnet``)
+         #. Set *Subnet* to *default (10.0.0.0/24)*
+         #. Set *Public IP* to the server's FQDN (e.g. ``ocp42.open-contracting.org``)
+
+         #. If not using Docker, set *NIC network security group* to *None*.
+
+         #. If using Docker, set *NIC network security group* to *Advanced*
+
+            #. Click *Create new*
```

Comment:
Is it necessary to create a new one, or can the same network security group be reused? (e.g. we can define a "web" group that allows 22, 80, 443, ICMP and Prometheus, like below)

Reply:
You aren't using generic network security groups elsewhere so we have followed your existing processes.

These are network security group's so they are already reused when a subnet is reused (they are per subnet, not per instance). 

If a server already exists inside the same subnet then you can reuse the same network security group for this subnet.